src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppElasticBeanstalkWindows.recipe

```diff
@@ -291,6 +291,34 @@
             "AdvancedSetting": true,
             "Updatable": true
         },
+        {
+            "Id": "LoadBalancerScheme",
+            "Name": "Load Balancer Scheme",
+            "Category": "LoadBalancer",
+            "Description": "Specify \"Internal\" if your application serves requests only from connected VPCs. Public load balancers serve requests from the Internet.",
```

Comment:
nit: add quotes around "public" and make both "Internal" and "Public" lowercase to indicate what the actual values are.

Reply:
Added quotes around "Public" via https://github.com/aws/aws-dotnet-deploy/compare/728ba2390e178d1ed44c2a832458ddcefee621e9..69845e4684d83f12f2ba998f5abb889dbbc86897